src/main/java/org/apache/commons/math4/util/MathArrays.java

```diff
@@ -823,10 +838,11 @@ public int compare(PairDoubleInteger o1,
      * @param to Final index of the range to be copied, exclusive. (This index may lie outside the array.)
      * @return the copied array.
      */
-    public static double[] copyOfRange(double[] source, int from, int to) {
+    @SuppressWarnings("index:argument.type.incompatible") // #1: As FastMath.min(len, source.length - from)) is the minimum of both the arguments, the expression <= source.length - fron and <= output
```

Comment:
This part of the comment: "the expression <= source.length - fron and <= output" is very unclear - what does "the expression" refer to? There's also a typo: "fron" should be "from".